manifest: sdk-connectedhomeip: Update revision (#18910)
nrfconnect/sdk-nrf: doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

Conversely, LIT devices are designed to be used for sensors or light switches, devices that only report data and are not controllable.
In such scenarios, the LIT device initiates communication and it is not able to answer with a small latency, but it can sleep for extended periods of time and achieve much lower power consumption than an SIT.

The LIT device starts operation in the SIT mode and remains in this state until the first ICD client registers to it.
This is necessary because the device is not responsive in the LIT mode, so client registration would be difficult.

> Once the ICD client is registered, the ICD device switches to LIT mode in order to save the energy.

Reviewer: ```suggestion
Once the ICD client is registered, the ICD device switches to LIT mode in order to save energy.
```

* :kconfig:option:`CONFIG_CHIP_ICD_UAT_SUPPORT` to enable the User Active Mode Trigger support.
  The User Active Mode Trigger allows triggering the ICD device to move from the idle to active state and make it immediately responsive, for example to change its configuration.
  This option is by default enabled for the LIT device.
* :kconfig:option:`CONFIG_CHIP_ICD_SIT_SLOW_POLL_LIMIT` to limit the slow polling interval value while the device is in the SIT mode.

>   This option can be used by the LIT device, to limit the slow poll interval used while temporarily working in the SIT mode.

Reviewer: ```suggestion
  This option can be used to limit the slow poll interval of an LIT device while temporarily working in the SIT mode.
```

> * :kconfig:option:`CONFIG_CHIP_ICD_DSLS_SUPPORT` to enable Dynamic SIT LIT switching support (DSLS).

Reviewer: ```suggestion
* :kconfig:option:`CONFIG_CHIP_ICD_DSLS_SUPPORT` to enable Dynamic SIT LIT switching (DSLS) support.
```

>   The DSLS allows the application to dynamically switch between SIT and LIT modes, as long as the requirements for these modes are met.

Reviewer: ```suggestion
  The DSLS support allows the application to dynamically switch between SIT and LIT modes, as long as the requirements for these modes are met.
```
